tests/unit/test_client.py

```
@@ -0,0 +1,357 @@
+import pytest
+from unittest.mock import Mock, patch, MagicMock, call
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)